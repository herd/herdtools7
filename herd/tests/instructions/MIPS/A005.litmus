MIPS A005

(* test ld instruction *)
{ int64_t 0:a1 = x; int64_t x=1}

P0;
 ld a0, 0(a1);

forall 0:a0 = 1
