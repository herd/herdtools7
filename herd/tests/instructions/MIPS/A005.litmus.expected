Test A005 Required
States 1
0:$4=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:$4=1)
Observation A005 Always 1 0
Hash=d33fef1633282273cf65bd2cad3b64fb

